app/controllers/storage/FlashKeys.scala

```diff
@@ -18,4 +18,5 @@ package controllers.storage
 
 object FlashKeys {
   val MOVEMENT_TYPE = "MOVEMENT_TYPE"
+  val UCR = "UCR"
```

Comment:
A minor thing, just to have one file less... why not moving `object Flashkeys` in FlashExtractor.scala as `object FlashExtractor`?